src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -417,6 +424,135 @@ bool emberAfOperationalCredentialsClusterUpdateNOCCallback(EndpointId endpoint,
     return true;
 }
 
+bool emberAfOperationalCredentialsClusterCertChainRequestCallback(EndpointId endpoint, app::CommandHandler * commandObj,
```

Comment:
This function name could probably be shortened to `emberAfOpCredsClusterCertChainReqCallback` if you are allowed to manually name it.

Reply:
does not seem possible